src/types.ts

```diff
@@ -889,17 +889,17 @@ export type DndOptionsType = {
    * Callback(targetNode, data)
    * @default null
    */
-  dragOver: null;
+  dragOver: null | ((e: WbNodeEventType & { event: DragEvent }) => void);
   /**
    * Callback(targetNode, data), return false to prevent autoExpand
    * @default null
    */
-  dragExpand?: null | ((e: WbNodeEventType & { event: DragEvent }) => void);
+  dragExpand?: null | ((e: WbNodeEventType & { event: DragEvent }) => boolean);
   /**
    * Callback(targetNode, data)
    * @default null
    */
-  drop?:
+  dragDrop?:
```

Comment:
We should leave the name as `drop` 
If we really want change anything, then it should be 'dragDrag' to 'drag' (See https://developer.mozilla.org/en-US/docs/Web/API/HTML_Drag_and_Drop_API )
Note that changing also requires updating our docs.

Reply:
only thought to rename it cause of dragDrag. i now rename dragDrag to only drag